assets/src/blocks/Timeline/Timeline.js

```diff
@@ -16,10 +16,10 @@ export const Timeline = (props) => {
 	const timelineNode = useRef(null);
 
 	const [stylesLoaded, stylesError] = useStyleSheet(
-    `https://cdn.knightlab.com/libs/timeline3/${TIMELINE_JS_VERSION}/css/timeline.css`
+		`https://cdn.knightlab.com/libs/timeline3/${TIMELINE_JS_VERSION}/css/timeline.css`
```

Comment:
Just checking if this spacing change is indeed intended?

Reply:
Excellent question :grin: I realized that file was indented with tabs and my code had spaces in it so I converted the whole file to tabs for coherence. I know that the local `.editorconfig` file defines the indentation as 2 spaces, I chose the lesser diff for now; I can also do the opposite if you're ok with it.

Reply:
Since we need to revisit our style config anyway, I guess it's ok to leave the file inconsistent if that means we don't need to modify more lines than are actually changing. Spacing changes can be an obstacle hiding more relevant commit messages.